### PR TITLE
feat(chat): change chat and marketplace buttons placement (Issue #2641)

### PR DESCRIPTION
**Pull request by Derikyan**
**Description:**

Show "New conversation"(Chat) and "Back to Chat"(Marketplace) buttons in header on all devices

Issues:

- Issue #2641 

**UI changes**

Снимок экрана 2024-11-25 в 11 59 09
Снимок экрана 2024-11-25 в 11 59 39

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Derikyan** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/540864)
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

File: apps/chat-e2e/src/ui/webElements/header.ts [hunk 1]
```diff
@@ -23,4 +24,16 @@ export class Header extends BaseElement {
   public rightPanelToggle = this.getChildElementBySelector(
     HeaderSelectors.rightPanelToggle,
   );
+
+  public newEntityButton = this.getChildElementBySelector(
+    SideBarSelectors.newEntity,
```
Comment (irinakartun) [inline apps/chat-e2e/src/ui/webElements/header.ts]: please, move the locator to `HeaderSelectors`

Comment (nartovm) [reply apps/chat-e2e/src/ui/webElements/header.ts]: done

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by irinakartun — DISMISSED** [review]

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by nartovm** [comment]
> /deploy-review
> 
> > Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
> > E2E tests status: failed

Tests are considered to be passed, there are 3 failed tests due to the known problem, investigation is in progress

File: apps/chat/src/components/Header/CreateNewChat.tsx [hunk 2]
```diff
@@ -20,7 +22,7 @@ interface Props {
   iconSize: number;
 }
 
-export const CreateNewChatMobile = ({ iconSize }: Props) => {
+export const CreateNewChat = ({ iconSize }: Props) => {
```
Comment (IlyaBondar) [inline apps/chat/src/components/Header/CreateNewChat.tsx]:
```suggestion
export const CreateNewConversation = ({ iconSize }: Props) => {
```

File: apps/chat/src/components/Header/CreateNewChat.tsx [hunk 3]
```diff
@@ -30,29 +32,47 @@ export const CreateNewChatMobile = ({ iconSize }: Props) => {
...
             size={iconSize + 6}
-            className="cursor-pointer text-secondary"
+            className="cursor-pointer text-secondary md:px-2"
           />
         ) : (
-          <div className="flex items-center justify-center rounded bg-accent-secondary-alpha p-[3px]">
+          <div
+            className={classNames(
+              'flex items-center justify-center rounded border border-transparent bg-accent-secondary-alpha p-[2px] md:px-[10px]',
+              messageIsStreaming || isActiveNewConversationRequest
```
Comment (IlyaBondar) [inline apps/chat/src/components/Header/CreateNewChat.tsx]: `isActiveNewConversationRequest` is already excluded on line 55

File: apps/chat/src/components/Header/CreateNewChat.tsx [hunk 4]
```diff
@@ -30,29 +32,47 @@ export const CreateNewChatMobile = ({ iconSize }: Props) => {
...
+                ? 'cursor-not-allowed'
+                : 'cursor-pointer hover:border-accent-secondary',
+            )}
+          >
             <IconPlus
-              className="cursor-pointer text-accent-secondary"
+              className={classNames(
+                'text-accent-secondary',
+                messageIsStreaming || isActiveNewConversationRequest
```
Comment (IlyaBondar) [inline apps/chat/src/components/Header/CreateNewChat.tsx]: `isActiveNewConversationRequest` is already excluded on line 55

**Review by nartovm — DISMISSED** [review]

**Review by irinakartun — DISMISSED** [review]

**Comment by Derikyan** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2660.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — APPROVED** [review]

**Review by irinakartun — APPROVED** [review]

**Pull request merged**